xmtp_mls/src/client.rs

```diff
@@ -198,6 +200,8 @@ where
     }
 
     pub async fn register_identity(&self) -> Result<(), ClientError> {
+        log::info!("registering identity");
+        // TODO: Mark key package as last_resort in creation
```

Comment:
Think we can remove this TODO. Must have snuck in with a merge. The TODO was done a few weeks ago